test/admin-hpa/features/support/sql_db_helper.rb

```diff
@@ -321,6 +313,13 @@ def self.delete_pupils_not_taking_check
     result.do
   end
 
+
+  def self.set_pupil_attendance_via_school(school_id,code)
```

Comment:
@activemq Why is this direct DB access required?  This is directly supported by the front-end.

Reply:
@jon-shipley Im doing this because simply because it takes too long to remove each pupil from each school through the front end. Doing this in a DB call makes it a hell of alot quicker. So effectively whats going on is we remove all entries in 'pupilAttendance' and then use the above method to set the 'attendanceId' to null, to put the pupil back in to a Not started state.